brd-ios/breadwallet/src/Redesign/Scenes/Swap/Swap/SwapInteractor.swift

```diff
@@ -9,8 +9,23 @@
 //
 
 enum SwapErrors: Error {
+    
     case noQuote
     case general
+    case toSmall
+    
+    var errorMessage: String {
+        switch self {
+        case .noQuote:
+            return "No quote for currency pair."
+            
+        case .general:
+            return "Something went wrong"
+            
+        case .toSmall:
+            return "Entered crypto amount was to small. Has to be above 50 usd"
```

Comment:
too* USD*

if this is temporary, then ignore.

Reply:
this needs 2 be localized anyhow.. i just added them cause its way easier 2 debug the app this way